dist/index.html

```diff
@@ -0,0 +1,392 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Catalog</title>
+    <link rel="stylesheet" href="/main.48b3db1d.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  <script src="/main.48b3db1d.js"></script></head>
+  <body class="page-content">
+    <header class="header">
+      <a href="#home">
+        <img src="/logo.6023b87e.png" alt="logo Moyo" class="header__logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#laptops_computers" data-qa="nav-hover" class="nav__link">
+             Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__wrapper">
+        <div class="main__row">
+          <div class="card" data-qa="card">
+            <img src="/imac.7817668a.jpeg" alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)" class="card__image">
```

Comment:
I think - this line length more than 80)
```suggestion
            <img 
              src="/imac.7817668a.jpeg" 
              alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)" 
              class="card__image"
            >
```